cmd/run.go

```diff
@@ -122,20 +121,8 @@ func start(cliCtx *cli.Context) error {
 		log.Fatal(err)
 	}
 
-	st := newState(cliCtx.Context, c, l2ChainID, []state.ForkIDInterval{}, stateSqlDB, eventLog, needsExecutor, needsStateTree)
+	st, currentForkID := newState(cliCtx.Context, c, tmpEthMan, l2ChainID, stateSqlDB, eventLog, needsExecutor, needsStateTree, false)
```

Comment:
you should create the real etherman right after the state, the use the real etherman everywhere after this step